service/worker/scanner/executions/current_execution.go

```diff
@@ -99,6 +107,8 @@ func currentExecutionScannerManager(
 	params shardscanner.ScanShardActivityParams,
 	domainCache cache.DomainCache,
 ) invariant.Manager {
+	logger := zap.L()
```

Comment:
let's avoid using global logger. we don't do that anywhere else and it is better to keep it that way. instead you can pass logger as parameter